wiki/src/main/resources/XWiki/PDFViewerMacro.xml

```diff
@@ -243,7 +243,7 @@
       'trustedOrigins': $trustedOrigins,
       'withCredentials': $withCredentials
     }))
-    #set ($pdfviewerurl = $xwiki.getDocument('XWiki.PDFViewerMacro').getAttachmentURL('pdfviewer.zip'))
+    #set ($pdfviewerurl = $services.webjars.url('pdf-js/2.9.359/web/viewer.html'))
```

Comment:
This is the old API. The problem with this is that the version of the WebJar is hard-coded, which means we need to remember to update the code whenever we update the version in the POM. This is a pain. The new API is:

    $services.webjars.url('pdf-js', 'web/viewer.html')